net.sapples.LiveCaptions.json

```diff
@@ -0,0 +1,108 @@
+{
+    "app-id" : "net.sapples.LiveCaptions",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "livecaptions",
+    "finish-args" : [
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=fallback-x11",
+        "--env=LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/app/lib"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/onnx_root",
+        "*.la",
+        "*.a"
+    ],
+    "build-options" : {
+        "env" : {
+            "ONNX_ROOT" : "/app/onnx_root"
+        }
+    },
+    "modules" : [
+        {
+            "name" : "onnxruntime-x64",
```

Comment:
you should have one module with two different sources so that the build commands are not duplicated.